docs/dev-env.md

```diff
@@ -89,6 +89,9 @@ in your home directory with the following line:
 set -g default-command "${SHELL}"
 ```
 
+Tmuxinator tests run 2 lightning nodes. [Core Lightning](https://github.com/ElementsProject/lightning) which runs as a gateway to the federation, and [LND](https://github.com/lightningnetwork/lnd) which represents the rest of the Lightning network. These tests are conducted to simulate the process of sending and receiving transactions via Lightning network to and from Fedimint.
```

Comment:
This might not be be up to date anymore. Aren't we doing 2 * LND now?

Reply:
Oh, you mean both nodes are LND, and not 1 CL and  1 LND?